src/app/pages/maps/search-map/map/map.component.scss

```diff
@@ -0,0 +1,3 @@
+agm-map {
```

Comment:
let's make it equal to content height - input - header - footer, like on the other maps